engine/crates/integration-tests/src/lib.rs

```diff
@@ -26,7 +26,12 @@ thread_local! {
 
 pub fn runtime() -> &'static Runtime {
     static RUNTIME: OnceLock<Runtime> = OnceLock::new();
-    RUNTIME.get_or_init(|| Runtime::new().unwrap())
+    RUNTIME.get_or_init(|| {
+        tokio::runtime::Builder::new_current_thread()
+            .enable_all()
+            .build()
+            .unwrap()
```

Comment:
Because of the rusoto situation. We were using upstream in the engine workspace and our patched version in cli. Since our patched version is about wasm, it only works single threaded (send_wrapper crate).